docs/source/workflows/base/relax.rst

```diff
@@ -1,2 +1,287 @@
+
+  .. code:: python
+
+        input_generator.get_spin_types()
+
+* ``magnetization_per_site``. (Type: Python None or a Python list of floats).
+  An input devoted to the initial magnetization specifications.
+  It accepts a list where each entry refers to an atomic site in the structure. The quantity is
+  passed as the spin polarization in units of electrons, meaning the difference between spin up and spin down
+  electrons for the site. This also corresponds to the magnetization of the site in Bohr magnetons (μB).
+  The default for this input is the Python value None and, in case of calculations with spin, the
+  None value signals that the implementation should automatically decide an appropriate default initial magnetization.
+  The implementation of such choice is code-dependent and described in the supplementary material of the manuscript (doi)
+
+* ``reference_workchain.`` (Type: a previously completed ``RelaxWorkChain``, performed with the same code as the
+  ``RelaxWorkChain`` created by ``get_builder``).
+  When this input is present, the interface returns a set of inputs
+  which  ensure  that  results of the new ``RelaxWorkChain`` (to be run) can be directly
+  compared to the ``reference_workchain``. This is necessary to create,
+  for instance, meaningful equations of state.
+
+
+
+Relaxation outputs
+..................
+
+To allow direct comparison and cross-verification of the results, the outputs of
+``RelaxWorkChain`` are standardized for all implementations and are defined as follows:
+
+* ``forces``.
+  The final forces on all atoms in eV/Å.
+  (Type: an AiiDA ``ArrayData`` of shape N×3, where N is the number of atoms in the structure).
+
+* ``relaxed_structure``. The structure obtained after the relaxation. It is not returned if the relax_type is ‘none’.
+  (Type: AiiDA ``StructureData``).
+
+* ``total_energy``. The total energy in eV associated to the relaxed structure
+  (or initial structure in case no relaxation is performed).
+  In general, even for calculations performed with the same code, there is no guarantee to have comparable
+  energies in different runs if the numerical parameters determined by the input generator change
+  (because, for instance, structures with different volumes are passed). However, in combination with the
+  input argument ``reference_workchain``, energies from different relaxation
+  runs become comparable, and their energy difference is well defined. (Type: AiiDA ``Float``).
+
+* ``stress``.   The final stress tensor in eV/Å^3.
+  Returned only when a variable-cell relaxation is performed.
+  (Type: AiiDA ``Float``).
+
+* ``total_magnetization``. The total magnetization in
+  μB (Bohr-magneton) units.  Returned only for magnetic calculations.
+  (Type: AiiDA ``Float``).
+
+
+CLI options
+...........
+
+The use of the CLI for the submission of a common workflow is reported in the :ref:`main page <how-to-submit>` of this documentation.
+For the relaxation workflow:
```

Comment:
Okay to hard-code this for now, but from experience this is a pain to maintain. Everytime the CLI is changed, this has to be probably updated and that is easy to forget. For `aiida-core` I implemented a pre-commit hook that will automatically update this. Just mentioning this so we can add an issue to add this at some point. Especially once we start documenting more CLI commands.